### PR TITLE
Update min versions for 1.27

### PR DESCRIPTION
**Pull request by johngmyers**

### CODE REVIEW

File: pkg/apis/nodeup/config.go [hunk 1]
```diff
@@ -197,7 +197,7 @@ func NewConfig(cluster *kops.Cluster, instanceGroup *kops.InstanceGroup) (*Confi
 }
 
 func UsesInstanceIDForNodeName(cluster *kops.Cluster) bool {
-	return cluster.Spec.ExternalCloudControllerManager != nil && cluster.IsKubernetesGTE("1.22") && cluster.Spec.GetCloudProvider() == kops.CloudProviderAWS
+	return cluster.Spec.ExternalCloudControllerManager != nil && cluster.Spec.GetCloudProvider() == kops.CloudProviderAWS
```
Comment (hakman) [inline pkg/apis/nodeup/config.go:200]: Do we still support in-tree CCM?

File: pkg/model/components/etcd.go [hunk 2]
```diff
@@ -42,11 +42,7 @@ func (b *EtcdOptionsBuilder) BuildOptions(o interface{}) error {
 		// Ensure the version is set
 		if c.Version == "" {
 			// We run the k8s-recommended versions of etcd
-			if b.IsKubernetesGTE("1.22") {
-				c.Version = DefaultEtcd3Version_1_22
-			} else {
-				c.Version = DefaultEtcd3Version_1_20
-			}
+			c.Version = DefaultEtcd3Version_1_22
```
Comment (hakman) [inline pkg/model/components/etcd.go:45]: Please also remove the `DefaultEtcd3Version_1_20` constant.

Comment (johngmyers) [reply pkg/model/components/etcd.go:45]: Can't. It's still referenced.

File: pkg/apis/nodeup/config.go (hunk 1 above)
Comment (johngmyers) [reply pkg/apis/nodeup/config.go:200]: Up to k8s 1.23, yes.

**Review by hakman — APPROVED** [review]

File: docs/releases/1.27-NOTES [hunk 3]
```diff
@@ -0,0 +1,34 @@
+## Release notes for kOps 1.27 series
```
Comment (rifelpet) [inline docs/releases/1.27-NOTES:1]: this should probably have the `.md` file extension for consistency

**Pull request merged**